modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -30,13 +38,19 @@ def show
 
         private
 
-        def set_notice_of_disagreement
-          return unless params[:nod_id]
+        def nod_id_present?
+          raise Common::Exceptions::ParameterMissing, 'nod_id' unless params[:nod_id]
+        end
 
+        def set_notice_of_disagreement
           @notice_of_disagreement ||= AppealsApi::NoticeOfDisagreement.find_by(id: params[:nod_id])
           raise Common::Exceptions::RecordNotFound, params[:nod_id] unless @notice_of_disagreement
         end
 
+        def validate_board_review_option
+          raise InvalidReviewOption unless @notice_of_disagreement.accepts_evidence?
```

Comment:
What does the JSON response for an invalid state here when this error is raised?

Reply:
i'm going to refactor this...considering a service object to manage some of the validation